docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: '3.8'
+
+services:
+  app:
+    build:
+      context: .
+      args:
+        NODE_VERSION: ${NODE_VERSION:-18}
+    ports:
+      - "${APP_PORT:-3000}:${APP_PORT:-3000}"
+    environment:
+      - NODE_ENV=development
+      - APP_PORT=${APP_PORT:-3000}
```

Comment:
It currently uses the `PORT` ([reference](https://github.com/fingerprintjs/dx-team-mock-for-proxy-integrations-e2e-tests/blob/main/src/index.ts#L8)) environment variable. I think we should use `PORT` instead of `APP_PORT` here, or vice versa in `src/index.ts`.

Reply:
Makes sense, updated APP_PORT as just PORT